controls/SV-235104.rb

```diff
@@ -64,21 +64,22 @@
     audit_admins = input('audit_admins').concat(["'rdsadmin'@'localhost'"])
 
 
-  query_audit_admins = %(
-    SELECT
-     * 
-  FROM
-     INFORMATION_SCHEMA.USER_PRIVILEGES 
-  where
-     PRIVILEGE_TYPE in 
-     (
-        'AUDIT_ADMIN',
-        'SUPER'
-     );
-  )
-
-  describe "List of configured audit admins" do
-    subject { sql_session.query(query_audit_admins).results.column('grantee') }
-    it { should match_array audit_admins }
-  end
+    query_audit_admins = %(
+      SELECT
+       * 
+    FROM
+       INFORMATION_SCHEMA.USER_PRIVILEGES 
+    where
+       PRIVILEGE_TYPE in 
+       (
+          'AUDIT_ADMIN',
+          'SUPER'
+       );
+    )
+  
+    describe "List of configured audit admins #{query_audit_admins}" do
```

Comment:
This will need to be updated to `describe "List of configured audit admins #{sql_session.query(query_audit_admins).results.column('grantee')}" do` which will lead to a test result that looks like:
<img width="903" alt="Screen Shot 2023-03-24 at 1 55 42 PM" src="https://user-images.githubusercontent.com/11844975/227603406-0472eda2-8964-4c57-957e-164e7d7caed1.png">

Would you like me to include the list of users in the describe line?

Reply:
We decided on not using the variable in the describe line